espn_api/football/offer.py

```diff
@@ -0,0 +1,64 @@
+from datetime import datetime
```

Comment:
I think this can be moved outside of football as `base_offer.py` so other sports can use it.